test/fixtures/render/issue-547.json

```diff
@@ -131,7 +131,7 @@
                               "content": "application/json"
                             }
                           },
-                          "content": "{}"
+                          "content": "{\n  \"\": [\n    \"foo\",\n    \"bar\"\n  ]\n}"
```

Comment:
I am wondering if this is correct. The underlying ADD is the following:

```apib
## GET /

+ Response 200 (application/json)
    + Attributes
        + (array)
            + foo
            + bar

```

There is a warning as the user has defined a property without a name and thus previously it was discarded. This logic is now proceeding with empty string.